### PR TITLE
Gives blood brothers better chat colouring.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Ghosts struggle to see which team is communicating on blood brothers gamemode. Makes each implant a unique colour so its easier to read.

## Why It's Good For The Game

Better readability for ghosts, nicer colours used.

image

Tested with other colours but I closed the tap.

## Changelog
:cl:
tweak: Makes each blood brother implant colour unique.
/:cl:

### CODE REVIEW

File: code/game/objects/items/implants/implant_bb.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	icon = 'icons/obj/radio.dmi'
 	icon_state = "headset"
 	item_color = "r"
+	var/implant_colour = "#ff0000"
```
Comment (Archanial) [inline code/game/objects/items/implants/implant_bb.dm:7]:
```suggestion
	var/implant_color = "#ff0000"
```
Bri'ish

Comment (PowerfulBacon) [reply code/game/objects/items/implants/implant_bb.dm:7]: british spellings are accepted

**Comment by heepox** [comment]
Good

File: code/modules/antagonists/brother/brother.dm [hunk 2]
```diff
@@ -96,9 +97,15 @@
 /datum/team/brother_team
 	name = "brotherhood"
 	member_name = "blood brother"
+	var/team_id
 	var/meeting_area
 	var/static/meeting_areas = list("The Bar", "Dorms", "Escape Dock", "Arrivals", "Holodeck", "Primary Tool Storage", "Recreation Area", "Chapel", "Library")
 
+/datum/team/brother_team/New(starting_members)
+	. = ..()
+	var/static/blood_teams
```
Comment (qwertyquerty) [inline code/modules/antagonists/brother/brother.dm:106]: Does making a static variable here do anything at all?

Comment (PowerfulBacon) [reply code/modules/antagonists/brother/brother.dm:106]: yes, its static

File: code/game/objects/items/implants/implant_bb.dm (hunk 1 above)
Comment (utilbird) [reply code/game/objects/items/implants/implant_bb.dm:7]: Godspeed

**Review by utilbird — APPROVED** [review]

**Pull request merged**